samples/oracle/InsertGplInfo.groovy

```diff
@@ -50,12 +50,12 @@ def alreadyLoaded(platform) {
 def insertGplInfo(options) {
   sql = DatabaseConnection.setupDatabaseConnection()
   sql.execute(
-      "INSERT INTO deapp.de_gpl_info(platform, title, organism, marker_type, genome_build, release_nbr)" +
-      " VALUES (:platform, :title, :organism, :marker_type, :genome_build, :release_number)",
+      "INSERT INTO deapp.de_gpl_info(platform, title, organism, marker_type, genome_build, gene_annotation_id)" +
+      " VALUES (:platform, :title, :organism, :marker_type, :genome_build, :gene_annotation_id)",
       [platform: options.platform, title: options.title,
               organism: options.organism, marker_type: options.marker_type,
               genome_build: options.genome_build ?: null,
-              release_number: options.release_number ?: null ]
+              gene_annotation_id: options.gene_annotation_id ?: null ]
```

Comment:
@forus  Genome annotation is the process of attaching biological information to sequences. This is done by many parties, resulting in many databases (e.g. Ensembl), which are updated regularly to reflect most recent insights. So the relation between a gene and a genomic location depends on which gene annotation and which genome build version is used.